pkg/postgres/postgres.go

```diff
@@ -51,7 +59,10 @@ func Wait(service, dsn string) *DB {
 	stream.Close()
 	// TODO(titanous): handle discoverd disconnection
 
-	db, err := Open(service, dsn)
+	// TODO(jpg): Race condition between DB registering with discoverd and being able
+	// to authenticate with password. Resolve this correctly before merging.
```

Comment:
I think a good option is to add `WaitForUserExistence` to https://github.com/flynn/flynn/blob/master/appliance/postgresql/client/client.go